<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressBookFrame</class>
 <widget class="QFrame" name="AddressBookFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Poppins</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font-size: 20px;
font-family: Poppins;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#000000;&quot;&gt;PK Address Book&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_copyAddressButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>171</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(170, 0, 0);
font-size: 13px;
font-family: Poppins;</string>
       </property>
       <property name="text">
        <string>Copy Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_deleteAddressButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>171</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(170, 0, 0);
font-size: 13px;
font-family: Poppins;</string>
       </property>
       <property name="text">
        <string>Remove Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_addAddressButton">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>171</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(170, 0, 0);
font-size: 13px;
font-family: Poppins;</string>
       </property>
       <property name="text">
        <string>Add New Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_backButton">
       <property name="maximumSize">
        <size>
         <width>171</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(170, 0, 0);
font-size: 13px;
font-family: Poppins;</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="m_addressBookView">
     <property name="minimumSize">
      <size>
       <width>990</width>
       <height>560</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QHeaderView::section {
	background-color: rgb(220, 220, 220);
	color: rgb(0, 0, 0);
	font-family: Poppins;
    font-size: 12px;
    border: 0px solid white;
	border-bottom: 1px solid black;
   height: 35px;
}

QTreeView::item
{
    font: 10px;
	color: rgb(0, 0, 0);
    height: 30px;
}

QTreeView::branch:selected {
	background-color: rgb(170, 0, 0);
	color: black;
}

QWidget:item:selected {
	background-color: rgb(220, 220, 220);
	color: rgb(0, 0, 0);
}

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="indentation">
      <number>5</number>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="headerShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_addAddressButton</sender>
   <signal>clicked()</signal>
   <receiver>AddressBookFrame</receiver>
   <slot>addClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_deleteAddressButton</sender>
   <signal>clicked()</signal>
   <receiver>AddressBookFrame</receiver>
   <slot>deleteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_copyAddressButton</sender>
   <signal>clicked()</signal>
   <receiver>AddressBookFrame</receiver>
   <slot>copyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addressBookView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>AddressBookFrame</receiver>
   <slot>addressDoubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_backButton</sender>
   <signal>clicked()</signal>
   <receiver>AddressBookFrame</receiver>
   <slot>backClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1198</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addClicked()</slot>
  <slot>backClicked()</slot>
  <slot>deleteClicked()</slot>
  <slot>copyClicked()</slot>
  <slot>addressDoubleClicked(QModelIndex)</slot>
 </slots>
</ui>
